src/Polly.RateLimiting/RateLimiterRejectedException.cs

```diff
@@ -96,14 +149,8 @@ public override void GetObjectData(SerializationInfo info, StreamingContext cont
     {
         Guard.NotNull(info);
 
-        if (RetryAfter.HasValue)
-        {
-            info.AddValue("RetryAfter", RetryAfter.Value.TotalSeconds);
-        }
-        else
-        {
-            info.AddValue("RetryAfter", -1.0);
-        }
+        info.AddValue(nameof(TelemetrySource), TelemetrySource ?? "(null)/(null)/(null)");
+        info.AddValue(nameof(RetryAfter), RetryAfter.HasValue ? RetryAfter.Value.TotalSeconds : -1.0);
```

Comment:
Not to do with your changes, but I wonder if reading and writing ticks rather than total seconds would be better in terms of resolution?